src/app/interfaces/course.interface.ts

```diff
@@ -1,8 +1,18 @@
-export interface ICoursePage {
+export interface ICoursePage extends IAddCourseData {
```

Comment:
if you inharite anything NEVER duplicate properties
the only one case is when you need change something (mostly relate to methods)